### DISCUSSION TITLE
Track XP per map ?

### DISCUSSION THREAD

**Discussion by GrimDawn-max** [Lailloken/Exile-UI#113] [Ideas & Feedback]
Would it be possible to track the XP gained per map, either as part of the map or loot stats already collected ? It can be done manually of course, but that's tedious and messy since I have to remember to keep track of pre-map XP and post map XP. It's not part of the Client.txt of course and if it involves OCR it's a non-starter but maybe there is another way to capture it. Here's hoping ...

**Comment by Lailloken** [comment]
Someone requested an exp overlay in the past (because other exp-tracking tools rely on being in the top-X on the leaderboard), and I think screen-reading the exp gauge/bar might be possible. My issue (until now) was that it's use-case is quite niche but requires a lot of setup and its own section in the settings menu -- which will become kind of bloaty if every small feature has its own section. But this might be solved by implementing it into the mapping tracker as an optional feature. I'll add this idea to the backlog for now, and I'll also have to check if the game renders the exp bar correctly (and granularly enough) to get an accurate reading.
